src/librados/librados.cc

```diff
@@ -4726,6 +4750,26 @@ extern "C" int rados_aio_read_op_operate(rados_read_op_t read_op,
   return retval;
 }
 
+extern "C" int rados_cache_pin(rados_ioctx_t io, const char *o)
+{
+  tracepoint(librados, rados_cache_pin_enter, io, o);
```

Comment:
if you add tracepoints, they need to get defined in the .tp file.  build with LTTNG (--with-lttng?)